src/main/java/robot/subsystems/elevator/commands/JoystickElevatorCommand.java

```diff
@@ -0,0 +1,65 @@
+package robot.subsystems.elevator.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import robot.Robot;
+import robot.subsystems.elevator.Elevator;
+
+/**
+ *
+ */
+public class JoystickElevatorCommand extends Command {
+    /**
+     * The Y value area in which the xbox joystick won't make the lift move.
+     */
+    private static final double XBOX_JOYSTICK_DEAD_BAND = 0;
```

Comment:
Move all constants in this file (and all constants in other files, if I missed them) to the Constants file.

Reply:
Do i open a global constants file then?
these arent the type of things that should be put in the subsystem-specific constants file